examples/sim_params.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I agree that there's merit in splitting out "simulation parameters" from "topology of the network", but I do not think that a pull request implementing the Raman feature is a good place to change this. There's the `SpectralInformation` stanza in the `example_network.json` which already contains these "global" information. I would prefer that structure to be used instead of a new `sim_params.json`.

Reply:
I made a mistake here, I missed the fact that `SpectralInfromation` is currently defined in the equipment library, not in the network topology JSON file. Sorry for my confusion. Let's take this and merge it as an improvement.

(I'll blame this on my recent YANG work where this is different.)